h/schemas/annotation.py

```diff
@@ -311,3 +311,82 @@ def _target_selectors(targets):
         return targets[0]['selector']
     else:
         return []
+
+
+class SearchParamsSchema(JSONSchema):
+    """Validates a JSON payload for an elasticsearch query via API"""
+    schema = {
+        # 'type': 'object',
+        'properties': {
+            '_separate_replies': {
+                'type': 'string',
+                'description': "Return a seperate set of annotations and their replies.",
+            },
+            'sort': {
+                'type': 'string',
+                'enum': ["created", "updated", "group", "id", "tags_raw", "raw_user", "uri"],
```

Comment:
No it doesn't work so I'll remove it. To sort by user es requires the field be non-analyzed so we must use the user_raw field but it can be exposed as user and translated to user_raw before querying.